fred2/sexp_tree.cpp

```diff
@@ -1574,11 +1573,7 @@ void sexp_tree::right_clicked(int mode)
 		for (j=0; j<(int)Operators.size(); j++) {
 			z = 0;
 			if (m_mode == MODE_CAMPAIGN) {
-				if (Operators[j].value & OP_NONCAMPAIGN_FLAG)
-					z = 1;
-
-			} else {
-				if (Operators[j].value & OP_CAMPAIGN_ONLY_FLAG)
+				if (!usable_in_campaign(Operators[j].value))
 					z = 1;
```

Comment:
This seems off. You've disabled noncampaign flags in campaigns, but no longer disable campaign only flags out-of-campaign. Is that intended?

Reply:
Yes, because currently there *are* no campaign-only sexps.  I figured if we ever need them, we can defer this until that time.